update_product_stream/README.md

```diff
@@ -22,12 +22,33 @@ be downloaded and extracted.
 
 1. Download the CSM software release tarball.
 
+> ***NOTE:*** CSM does NOT support the use of proxy servers for anything other than downloading artifacts from external endpoints.
```

Comment:
I think this may need to be indented more.  "Extract the release distribution" should be step 2 but it renders as step 1 again.   I believe that is because of the indenting here.